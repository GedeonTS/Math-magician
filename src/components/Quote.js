import React from 'react';

function Quote() {
  return (
    <section className="quote-section">
      <p>
        Mathmatics is not about numbers,equations, computations,
        or algorithms: it is about understanding.
        <span className="author">-William Paul Thurston</span>
      </p>
    </section>
  );
}

export default Quote;
