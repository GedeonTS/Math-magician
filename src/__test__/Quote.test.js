import React from "react";
import renderer from 'react-test-renderer';
import Quote from '../components/Quote';
import '@testing-library/jest-dom';

describe('Test of Quote page', () => {
    it('test if Quote page is rendered', () => {
        const quoteTest = renderer
            .create(<Quote />)
            .toJSON();
        expect(quoteTest).toMatchSnapshot();
    });
});
