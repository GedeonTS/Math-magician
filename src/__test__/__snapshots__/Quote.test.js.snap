// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test of Quote page test if Quote page is rendered 1`] = `
<section
  className="quote-section"
>
  <p>
    Mathmatics is not about numbers,eauations, computations, or algorithms: it is about understanding.
    <span
      className="author"
    >
      -William Paul Thurston
    </span>
  </p>
</section>
`;
